client/a8c-for-agencies/sections/marketplace/checkout/submit-payment-info.tsx

```diff
@@ -67,9 +67,14 @@ export default function SubmitPaymentInfo( { disableButton }: { disableButton?:
 		if ( status === 'success' ) {
 			dispatch( recordTracksEvent( 'calypso_a4a_client_checkout_submit_payment_info_success' ) );
 			dispatch(
-				successNotice( translate( 'Thank you for your purchase!' ), {
-					displayOnNextPage: true,
-				} )
+				successNotice(
+					paymentMethodRequired
```

Comment:
I'm not sure why we need to use the `paymentMethodRequired` flag here. Can we just always use the `Thank you for your purchase! Your agency can now set up your product.` text here?

Reply:
Good catch. I was thinking to use isClient and I don't why I used that variable. I just updated it.